lib/client/client.go

```diff
@@ -663,13 +717,32 @@ func (proxy *ProxyClient) DeleteAppSession(ctx context.Context, sessionID string
 	return nil
 }
 
-// GetDatabaseServers returns all registered database proxy servers.
-func (proxy *ProxyClient) GetDatabaseServers(ctx context.Context, namespace string) ([]types.DatabaseServer, error) {
+// FindDatabaseServersByFilters returns all registered database proxy servers.
```

Comment:
```suggestion
// FindDatabaseServersByFilters returns registered database proxy servers that match the provided filter.
```

Technically it doesn't return *all* database servers.